edgexfoundry/edgex-compose
# build: Update to using eKuiper 1.4 (unpinned to patch)

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:



## Testing Instructions
Run edgex stack
Create eKuiperStream and Rule
Verify via eKuiper logs that it was successful in securely connecting to EdgeX MessageBus.
